airbyte-integrations/connectors/source-hubspot/source_hubspot/streams.py

```diff
@@ -379,11 +379,17 @@ def read_records(
         except requests.exceptions.HTTPError as e:
             raise e
 
+    def error_message(self, response: requests.Response) -> str:
+        return self.parse_response_error_message(response)
+
     def parse_response_error_message(self, response: requests.Response) -> Optional[str]:
         try:
             body = response.json()
         except json.decoder.JSONDecodeError:
-            return response.text
+            message = response.text
+            if len(message) > 100:
```

Comment:
Where does this 100 characters limitation comes from? If it's a UI issue, why does the source needs to know about it?

Reply:
@maxi297 that's fair. The thing is the issue has a linked PR that fixes it on the UI and the issue is still not closed, so I thought we need to handle it here as well. Do you think we can just close the issue?

Reply:
If this change should be done in the UI because it is just UI related, yes I would close this one and push for the PR on the UI side to be closed